### PR TITLE
Web strings reworked 3

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

**Comment by mitchdowney** [comment]
@comradekingu actually this is an easy one.

1) "Show Notes" is an expression common in American parlance in podcasting. I get that it's unclear though what "Show notes" means to translators, because it sounds like a verb. Is there a way we can note within Weblate that the "Show notes" are the "notes/description" a podcaster assigns to a specific episode? [Sample of "Show notes" usage](https://castos.com/podcast-show-notes/)

2) Using the term "open source" where we are using it is a marketing decision. None of our family and friends in America know what "Libre software" means, and they are our primary audience. The apps in the Podcasting 2.0 community are using "open source" as the term as well. I'm going to reject PRs that contain that change. Happy to use "Libre software" for non-English speakers, as I don't know what is commonly understood by speakers in other parts of the world. If that can be noted in Weblate, then let's add that.

2 continued) I like sprinkling in "Libre software" though. I see it as an educational opportunity to spur curiosity in why this "open source" app is also calling itself "libre software". There are just specific points in the app where we opt for "open source" for marketing purposes. Specifically we want "open source" for these lines.

`"features - support open source software": "Support open source software",`

`"open source": "Open Source",`

If this "open source" debate is a sticking point I respect your difference in opinion. Our preference is to appeal to what the people we know already understand (open source) and then give them an opportunity to learn that we are *more* than open source, we're libre software. I hope you'll accept our decision, but if not please let me know.

File: public/locales/en/common.json [hunk 1]
```diff
@@ -296,9 +296,9 @@
   "Share": "Share",
   "Share Clip": "Share Clip",
   "Show all podcasts": "Show all podcasts",
-  "Show full screen player": "Show full screen player",
+  "Show full screen player": "Show full-screen player",
   "Show navigation menu": "Show navigation menu",
-  "Show Notes": "Show notes",
+  "Show Notes": "Show-notes",
```
Comment (comradekingu) [inline public/locales/en/common.json:301]:
```suggestion
  "Show Notes": "The show notes",
```

**Comment by comradekingu** [comment]
@mitchdowney I have tried every possible way I can to connect to the XMPP server.
Are you in it during European hours?

It isn't my decision not to accept, but lets see if we are on the same page before selling anyone else on the concept.

I posit none of your normie-friends know what the trademark "Open Source" actually means, as defined by the OSI.
That is to say people not already in the know.
This is because the term purposefully is misleading, as it focuses on everything but the freedoms granted.
Almost without fail, people answer "gratis", "freeware", or "no cost" about its meaning, and that is because it failed to differentiate itself in the minds of people as something more valuable.

The business term "Open source" enjoys common familiarity largely just as much as it is effectively a misunderstanding.
By design, there is no way of getting to "understand" it from a familiarity of knowing the term, or the meaning of its components.

"Libre" isn't meant to sprinkle, any more than "free" is purposefully avoided where it can to avoid it being misinterpreted as respectively gratis or libre when not.
One word, throughout, meaning one thing, that can be deciphered from its general meaning, with further description added where natural, has to be good.

In searches and the greater corpus, "free software" and "open source" are vastly more popular, which is why I put that in the searchable app description.
In what is marketably communicative, older terms and the ability to find something when having to search in that domain is only a given that doesn't necessitate better understanding.
Without a better understanding, I don't think a better response is likely…?

For libre software that relies on user donations, it is my experience that "libre" works better.
Granted it is something I do a lot of, but take a look at the top Librepay projects for example.
A unified strategy always brings in more donations, and "copylefted libre" is the one that works if it is paired with the explanation that is already there.
It also does translate better for reasons why "libre" is easier to actually understand, seeing as direct translations of "open source" equate to ~"source that is open", which is not different from non-libre.
To be specific, only (?) English has this duality, but other languages could go from "frei" "frei Software" to say ~"offen quellcode" similar to "quelloffen". Alternatively it says just "Open Source" in other languages, meaning untranslated.

Similarly, "copylefted" qualifies the libre, so not as to sell the whole concept short.
The "what am I supporting, really" is important, and strong copyleft is more important to a consumer than it is to a business
capable and seeking to employ its own control of something. It also is a lot harder to have nefarious intentions that way.
That is why I made it "- with all" btw.

I think the "Supports ____" category in the comparison should be _copylefted libre_, and that "member" should have a bigger smiley.
It should also IMO be listed at the top, as opposed to the bottom.
All psychology, but still.

(I don't know where `"open source": "Open Source",` is used.)

The Podcasting 2.0 apps or the general concept doesn't rely on uniformity in under-communicating freedom.
I dare say it is my hope to do somewhat better than the general linguistic mastery on display.
To set "Podverse" apart from the rest is a gamble that has no downside that I can see.
Someone has to be first in managing to communicate well what sets it all apart from non-libre apps and ecosystems.
I listen to the "Podcasting 2.0" podcast, and that last stepping stone grinds my goat every time it is mentioned.

File: public/locales/en/common.json [hunk 2]
```diff
@@ -296,9 +296,9 @@
   "Share": "Share",
   "Share Clip": "Share Clip",
   "Show all podcasts": "Show all podcasts",
-  "Show full screen player": "Show full screen player",
+  "Show full screen player": "Show full-screen player",
   "Show navigation menu": "Show navigation menu",
-  "Show Notes": "Show notes",
+  "Show Notes": "The show notes",
```
Comment (comradekingu) [inline public/locales/en/common.json:301]: Just avoiding disaster here.

**Comment by mitchdowney** [comment]
@comradekingu alright, this is well-explained and I think I basically understand where you're coming from. This is a pretty deep discussion, and I don't have a change of strategy for the more marketing focused usages of "open source" right now. I've been personally attached to "open source" software since the mid-90s, and it won't be easy for me to stop thinking of it favorably, or that we're selling ourselves short if people think we're "only open source".

I can't really defend my take on this beyond it's a gut-feeling for me that "open source" has more significance to people in my life, and the more important thing to me is Podverse _is_ copylefted libre software more so than it's something we put front-and-center in our marketing.

I hope using "libre" and "copylefted libre" throughout the apps for the less marketing focused sections is at least an improvement over wholesale use of "open source". I'll keep thinking about the distinction and will try to stay open to changing my mind someday.

**Comment by mitchdowney** [comment]
Closing this PR as I think your other changes were already included in reworked 4? Sorry if some changes you wanted are missing and I closed this in error.

**Pull request closed without merging**